src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSessionManager.cs

```diff
@@ -228,6 +249,53 @@ Task DisposeCoreInternalAsync()
         }
     }
 
+    internal async Task<bool> RebuildProjectAsync(
```

Comment:
nit: I think this is a "build", not a "rebuild". The former is incremental, while the latter is not.